src/base/bittorrent/session.cpp

```diff
@@ -3816,22 +3823,18 @@ bool Session::addMoveTorrentStorageJob(TorrentHandleImpl *torrent, const QString
         if (iter != m_moveStorageQueue.end()) {
             // remove existing inactive job
             m_moveStorageQueue.erase(iter);
+            LogMsg(tr("\"%1\" has cancelled to move storage from \"%2\" to \"%3\".").arg(torrent->name(), currentLocation, iter->path));
         }
     }
 
-    QString currentLocation = torrent->actualStorageLocation();
-    if (!m_moveStorageQueue.isEmpty() && (m_moveStorageQueue.first().torrentHandle == torrentHandle)) {
-        // if there is active job for this torrent consider its target path as current location
-        // of this torrent to prevent creating meaningless job that will do nothing
-        currentLocation = m_moveStorageQueue.first().path;
-    }
-
-    if (QDir {currentLocation} == QDir {newPath})
+    if (QDir {currentLocation} == QDir {newPath}) {
+        LogMsg(tr("\"%1\" has rejected to move storage to \"%2\".").arg(torrent->name(), newPath));
```

Comment:
Request to move "\"%1\" to its current location rejected?
Also is the word "storage" necessary here and in all other places in this PR?

Reply:
>Request to move ""%1" to its current location rejected?

Sorry, it's your proposal for this message format? Or something else?

Reply:
>Also is the word "storage" necessary here and in all other places in this PR?

"Move torrent" doesn't sound clear enough for me. But it's no problem to omit this word if everybody else understand it correctly.

Reply:
>Sorry, it's your proposal for this message format? Or something else?

The current format seems like a generic one that would include several variables (e.g. Reason: %3) but this string is only used here for one specific reason. Also nowhere in the current msg is it mentioned that the destination path is the same as the current.

>"Move torrent" doesn't sound clear enough for me

I would even omit "torrent"... it's a torrent client, is there something else it is moving?